Manager/Doctrine/AccessTokenManager.php

```diff
@@ -47,4 +47,13 @@ public function clearExpired(): int
             ->getQuery()
             ->execute();
     }
+
+    public function clearRevoked(): int
+    {
+        return $this->entityManager->createQueryBuilder()
+            ->delete(AccessToken::class, 'at')
+            ->where('at.revoked = 1')
```

Comment:
@franjo-zadelj-trikoder DQL supports [boolean literals](https://www.doctrine-project.org/projects/doctrine-orm/en/2.7/reference/dql-doctrine-query-language.html#literal-values):
```suggestion
            ->where('at.revoked = true')
```

This makes the code more consistent and easier to read IMO, @X-Coder264 your thoughts?